main.nf

```diff
@@ -39,6 +39,7 @@ def helpMessage() {
       --relevel                     Tsv indicating list of factors (conditions in the metadata table) and the new level on which to relevel the factor. Check contrasts docs.
       --logFCthreshold              Threshold (int) to apply to Log 2 Fold Change to consider a gene as differentially expressed.
       --genelist                    Txt file with list of genes (one per line) of which to plot heatmaps for normalized counts across all samples.
+      --nsubgenes                   Number (int) of genes to subset in the vst step of deseq. Reduce default if encounter errors with smaller datasets
```

Comment:
Thanks for the addition and contribution, in order to make it clearer what exactly this parameter edits, I would suggest a clearer name such as `vst_genes_number`. What do you think?

Reply:
Sure, sounds good. Thanks for reviewing this Gisela!